pkg/manager/member/tiflash_util.go

```diff
@@ -291,7 +291,7 @@ func setTiFlashCommonConfigDefault(config *v1alpha1.CommonConfig, externalCluste
 	setTiFlashProfilesConfigDefault(config.FlashProfile)
 }
 
-func setTiFlashFlashConfigDefault(config *v1alpha1.Flash, externalClusterName string, clusterName, ns string) {
+func setTiFlashFlashConfigDefault(config *v1alpha1.Flash, externalClusterName string, clusterName, ns, clusterDomain string) {
 	if config.TiDBStatusAddr == nil {
 		if len(externalClusterName) > 0 {
```

Comment:
The `externalClusterName` is added for the heterogeneous deployment, however, with across Kubernetes deployment, we should not set the address according to the `spec.cluster.name` only.